### ISSUE TITLE
Tests: Add accessible tests

### ISSUE THREAD

**Issue by jcfranco** [Esri/calcite-components#1159]
Breakout issue from https://github.com/Esri/calcite-components/pull/1153#event-3906164330

The above PR addressed some a11y issues and each component's e2e test should be updated to cover the fix:

- [x] ~~`calcite-accordion`~~
- [x] ~~`calcite-accordion-item`~~
- [x] ~~`calcite-action`~~ 
- [x] ~~`calcite-action-bar`~~
- [x] ~~`calcite-action-group`~~
- [x] ~~`calcite-action-pad`~~
- [x] ~~`calcite-avatar`~~
- [x]  ~~`calcite-alert`~~
- [x] ~~`calcite-block`~~ - @jcfranco
- [x] ~~`calcite-block-section`~~ - @jcfranco
- [x] `calcite-button` - @driskull
- [x]  ~~`calcite-card`~~
- [ ] `calcite-checkbox` - @eriklharper 
- [x] ~~`calcite-chip`~~
- [x] ~~`calcite-color`~~ - @jcfranco
- [x] `calcite-color-hex-input` - @jcfranco
- [x] `calcite-color-swatch` - @jcfranco
- [x] ~~`calcite-combobox`~~
- [x] ~~`calcite-combobox-item`~~
- [ ] `calcite-date` @paulcpederson 
- [ ] `calcite-date-day` @paulcpederson 
- [ ] `calcite-date-month` @paulcpederson 
- [ ] `calcite-date-month-header`@paulcpederson 
- [ ] `calcite-dropdown` - @macandcheese 
- [ ] `calcite-dropdown-group` - @macandcheese 
- [ ] `calcite-dropdown-item` - @macandcheese 
- [x] ~~`calcite-fab`~~
- [x] ~~`calcite-filter`~~ - @jcfranco
- [x] ~~`calcite-flow`~~
- [x] `calcite-graph` - @driskull
- [x] ~~`calcite-handle`~~ - @jcfranco
- [x] ~~`calcite-icon`~~ - @jcfranco
- [ ] `calcite-input` @macandcheese 
- [ ] `calcite-input-message` @macandcheese 
- [ ] `calcite-label` @eriklharper 
- [x] ~~`calcite-link`~~
- [ ] `calcite-loader` @paulcpederson 
- [ ] `calcite-modal` @paulcpederson 
- [x] ~~`calcite-notice`~~
- [x] ~~`calcite-option`~~ - @jcfranco
- [x] ~~`calcite-option-group`~~ - @jcfranco
- [x] ~~`calcite-pagination`~~
- [x] ~~`calcite-panel`~~
- [x] ~~`calcite-pick-list`~~ - @jcfranco
- [x] `calcite-pick-list-group` - @jcfranco
- [x] `calcite-pick-list-item` - @jcfranco
- [x] `calcite-popover` - @driskull 
- [x] `calcite-popover-manager` - @driskull 
- [x] `calcite-progress` - @driskull 
- [x] ~~`calcite-radio-button`~~ - @eriklharper 
- [x] ~~`calcite-radio-button-group`~~ - @eriklharper 
- [ ] `calcite-radio-group` - @eriklharper 
- [ ] `calcite-radio-group-item` - @eriklharper 
- [x] ~~`calcite-rating`~~
- [x] `calcite-scrim` - @driskull 
- [x] ~~`calcite-select`~~ - @jcfranco
- [x] ~~`calcite-shell`~~
- [x] ~~`calcite-shell-center-row`~~
- [x] ~~`calcite-shell-panel`~~
- [ ] `calcite-slider` - @eriklharper 
- [x] ~~`calcite-sortable-list`~~
- [ ] `calcite-split-button` - @kstinson14 
- [ ] `calcite-stepper` - @macandcheese 
- [ ] `calcite-stepper-item` - @macandcheese 
- [x] `calcite-switch` - @driskull 
- [x] ~~`calcite-tab`~~
- [x] ~~`calcite-tab-nav`~~
- [x] ~~`calcite-tab-title`~~
- [x] ~~`calcite-tabs`~~
- [x] ~~`calcite-tile`~~ - @eriklharper 
- [x] ~~`calcite-tile-select`~~ - @eriklharper 
- [x] ~~`calcite-tile-select-group`~~ - @eriklharper 
- [x] ~~`calcite-tip`~~ - @jcfranco
- [x] ~~`calcite-tip-group`~~ - @jcfranco
- [x] ~~`calcite-tip-manager`~~ - @jcfranco
- [x] `calcite-tooltip` - @driskull 
- [x] `calcite-tooltip-manager` - @driskull 
- [x] `calcite-tree` - @driskull 
- [x] `calcite-tree-item` - @driskull 
- [x] ~~`calcite-value-list-item`~~ - @jcfranco
- [x] ~~`calcite-value-list`~~ - @jcfranco

**Comment by macandcheese** [comment]
I can take card and dropdown.

**Comment by driskull** [comment]
I signed up for some. Some may already have a11y tests so we can just cross them out if necessary.

**Comment by driskull** [comment]
I've updated the list and crossed out the components that already have at least one `accessible` test. If you would like to create more complete tests feel free to do so. Otherwise, I think we can just concentrate on the ones missing tests.

**Comment by julio8a** [comment]
What's the status of this one?

**Comment by driskull** [comment]
I think we can close
